src/Microsoft.Diagnostics.Runtime/src/Desktop/V45Runtime.cs

```diff
@@ -695,5 +695,15 @@ internal override uint GetExceptionHROffset()
         {
             return PointerSize == 8 ? 0x8cu : 0x40u;
         }
+
+        public override string GetJitHelperFunctionName(ulong addr)
+        {
+            return _sos.GetJitHelperFunctionName(addr);
+        }
+
+        public override string GetMethodTableName(ulong addr)
+        {
+            return GetNameForMT(addr);
```

Comment:
Maybe consider just renaming GetNameForMT to be GetMethodTableName instead of carrying two functions?  I'm indifferent either way.  It's fine as is if you want to leave it.

Reply:
Fixed